packages/lesswrong/server/scripts/mergeAccounts.ts

```diff
@@ -79,12 +79,42 @@ Vulcan.mergeAccounts = async (sourceUserId: string, targetUserId: string) => {
   if (!sourceUser) throw Error(`Can't find sourceUser with Id: ${sourceUserId}`)
   if (!targetUser) throw Error(`Can't find targetUser with Id: ${targetUserId}`)
 
+  // DO NOT transfer LWEvents, there are way too many and they're probably not important after the fact
+  // await transferCollection({sourceUserId, targetUserId, collectionName: "LWEvents"})
```

Comment:
How do post reads work? I'm like 30% that it's through LWEvents, but I think we got another collection.

Reply:
It's in two places. The `ReadStatuses` collection records the current is-read status of things, can be edited with the mark-as-unread menu item, is used for highlighting, and is queried frequently in order to determine whether posts in lists should be bolded or dimmed. `LWEvents` records individual read-events, and is only queried if you use the slightly obscure "highlight new comments since <date>" menu, in order to provide a list of dates when you previously visited the post to choose from.

Reply:
@jimrandomh do you think it makes sense to merge LWEvents in this case? My guess is that they're too big a collection to move easily and the payoff isn't worth it. But doublechecking your read.

Reply:
I think I'd leave `LWEvents` alone, and edit `ReadStatuses`